projects/dotcms-js/src/lib/core/login.service.ts

```diff
@@ -18,6 +17,8 @@ export interface DotLoginParams {
     backEndLogin: boolean;
 }
 
+export const LOGOUT_URL = '/dotAdmin/logout';
```

Comment:
is it possible that the user would like to override this URL?

Reply:
Not at the moment is a SAML thing.